### ISSUE TITLE
Apple II GS null-modem serial cable (custom build) do not work properly

### ISSUE THREAD

**Issue by Dwlads** [ADTPro/adtpro#166]
Hi Guys,

First of all, Kudos for this fantastic project :)

I made a custom null-modem serial cable for my Apple II GS and looks like I did something wrong. I Followed the cable diagram available at adt.com/connectionsserial.html, as follows:

![ADTProNull8](https://user-images.githubusercontent.com/127326586/223759849-55651a45-1266-40d7-af75-b4d49624780b.png)

So my cable connections was made like this:

![connections](https://user-images.githubusercontent.com/127326586/223860868-76a05fbd-5966-4056-b50f-5da53bf3822e.jpeg)

My question is simple: Did I do something wrong? 

My solder skills are good, and I checked cable continuity, and everything looks fine, but I received a "Host Timeout" message when I tried to perform a (D)DIR on Adtpro 2.1.0.

Thank you and best regards.

**Comment by david-schmidt** [comment]
The short answer is, "it depends."  Assuming the pinout you're describing looks the same and has the same pin orientation as the pictures in https://adtpro.com/connectionsserial.html#MiniDIN8 , then the cable seems like it's reasonably well wired.  But there's a lot more variables in play.  The host computer's serial connection (whether native or USB-based), the COM port it's really using vs. the one you selected, the plug you used in the back of the GS... all are important to get right, and will have the exact same effect with "Host Timeout" if they're wrong.

**Comment by Dwlads** [comment]
Hi David,

As mentioned in our e-mail exchange, I finally had a successful cable connection. I used a better-quality Min-Din 8 connector and changed how I soldered the wires. Below is a simple scheme showing how I did this time:

![Connections Success](https://user-images.githubusercontent.com/127326586/225014263-2504ed6c-1924-47e1-819d-154525f525af.jpeg)

In addition, I hold the Min-Din 8 connector during the soldering process using Blu Tack to transfer the heat and avoid connector melting. 

So, this is the end of my custom cable saga, and again a BIG THANK YOU for your support and Kudos for this fantastic project.

Best regards.
